app/src/main/java/org/mifos/mobile/ui/home/HomeOldFragment.kt

```diff
@@ -128,6 +127,7 @@ class HomeOldFragment : BaseFragment(), OnRefreshListener {
         super.onResume()
         registerReceiver()
         activity?.invalidateOptionsMenu()
+        (activity as? BaseActivity)?.showToolbar()
```

Comment:
wont MFScaffold take care of this ?

Reply:
We dont have Scaffold implemented on Home Screen as of now, so have to explicitly tell to show toolbar.